detector/aws_alb_duplicate_name.go

```diff
@@ -9,22 +9,20 @@ import (
 
 type AwsALBDuplicateNameDetector struct {
 	*Detector
-	IssueType     string
-	TargetType    string
-	Target        string
-	DeepCheck     bool
 	loadBalancers map[string]bool
 }
 
 func (d *Detector) CreateAwsALBDuplicateNameDetector() *AwsALBDuplicateNameDetector {
-	return &AwsALBDuplicateNameDetector{
+	nd := &AwsALBDuplicateNameDetector{
 		Detector:      d,
-		IssueType:     issue.ERROR,
-		TargetType:    "resource",
-		Target:        "aws_alb",
-		DeepCheck:     true,
 		loadBalancers: map[string]bool{},
 	}
+	nd.Name = "aws_alb_duplicate_name"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_alb"
```

Comment:
__[gometalinter]__
* [goconst] 2 other occurrence(s) of "aws_alb" found in: aws_alb_invalid_subnet.go:24:14 aws_alb_invaid_security_group.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->